src/main/java/org/mifos/connector/slcb/camel/routes/file/FileProcessingRoute.java

```diff
@@ -0,0 +1,89 @@
+package org.mifos.connector.slcb.camel.routes.file;
+
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import org.mifos.connector.slcb.camel.routes.transfer.BaseSLCBRouteBuilder;
+import org.mifos.connector.slcb.dto.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import static org.mifos.connector.slcb.camel.config.CamelProperties.*;
+
+@Component
+public class FileProcessingRoute extends BaseSLCBRouteBuilder {
+
+    @Autowired
+    private CsvMapper csvMapper;
+
+    @Override
+    public void configure() {
+
+        /**
+         * Parse the [Transaction] array from the csv file
+         * exchangeInput: [LOCAL_FILE_PATH] the absolute path to the csv file
+         * exchangeOutput: [TRANSACTION_LIST] containing the list of [Transaction]
+         */
+        from("direct:get-transaction-array")
```

Comment:
not sure.
file routes will help us to read the data but I think here the file path is dynamic since it is downloaded from cloud. So there are two challenges.

File path is dynamic
We still need to process the file in CSV format since file route will read data in binary format.